Tools/PIDML/pidOnnxModel.h

```diff
@@ -213,48 +216,104 @@ struct PidONNXModel {
     }
   }
 
+  template <typename... P1, typename... P2>
+  static constexpr bool is_equal_size(o2::framework::pack<P1...>, o2::framework::pack<P2...>)
+  {
+    return sizeof...(P1) == sizeof...(P2);
+  }
+
+  static float scale(float value, const std::pair<float, float>& scalingParams)
+  {
+    return (value - scalingParams.first) / scalingParams.second;
+  }
+
+  template <typename C, typename T>
+  typename C::type getPersistentValue(const T& rowIterator)
```

Comment:
This also could replace the O2 function in ASOA.h, which now gets the value at the arrow level